lib/tuxedo.rb

```diff
@@ -82,7 +82,7 @@ def underscored_name
     # alias_method wont work well with inheritance/includes/extends
     #
     def setup_alias_method(name = underscored_name)
-      define_method(name) { object }
```

Comment:
Alright thanks that looks interesting. Using `alias_method` I get the following benchmark

```
Calculating -------------------------------------
     simpleDelegator   105.618k i/100ms
              tuxedo   114.589k i/100ms
           charlatan   109.471k i/100ms
-------------------------------------------------
     simpleDelegator      3.564M (± 8.5%) i/s -     17.744M
              tuxedo      4.563M (± 8.5%) i/s -     22.689M
           charlatan      4.611M (± 8.0%) i/s -     22.989M

Comparison:
           charlatan:  4611406.0 i/s
              tuxedo:  4563338.0 i/s - 1.01x slower
     simpleDelegator:  3563725.1 i/s - 1.29x slower
```

For me thats good enough, Rails way is a bit more complicated and not that readable. Thanks for your imput!